raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -128,100 +128,70 @@ def test_is_safe_to_wait():
 
 def test_next_route_amount():
     """ Routes that dont have enough available_balance must be ignored. """
-    amount = 10
     reveal_timeout = 30
     timeout_blocks = reveal_timeout + 10
     amount = UNIT_TRANSFER_AMOUNT
-    channel1 = factories.make_channel(our_balance=amount)
-    channel2 = factories.make_channel(our_balance=0)
-    channel3 = factories.make_channel(our_balance=amount)
 
-    channel_map = {
-        channel1.identifier: channel1,
-        channel2.identifier: channel2,
-        channel3.identifier: channel3,
-    }
+    channels = factories.make_channel_set([
+        {'our_state': {'balance': amount}},
```

Comment:
IMO, properties for channels are hard to remember and you have to check if what you want is supported.
Maybe introduce `ChannelProperties(balance=15)`?
This way, i can easily see the params available to me in my editor/IDE

Reply:
I made NamedTuples with the available parameters already, just the way it is implemented now they are hidden behind other functions, I also think it's awkward... I will come up with a better solution and make it part of the follow-up pr.